pygmt/helpers/caching.py

```diff
@@ -22,6 +22,7 @@ def cache_data():
         "@earth_mag_01d_g",
         "@earth_mag4km_01d_g",
         "@earth_mask_01d_g",
+        "@earth_mdt_01d_g",
```

Comment:
The `07m` data is not tiled.
```suggestion
        "@earth_mdt_01d_g",
        "@earth_mdt_07m_g",
```